src/authplus.ts

```diff
@@ -11,7 +11,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Compute, GoogleAuth, JWT, OAuth2Client} from 'google-auth-library';
+import {
+  Compute,
+  GoogleAuth,
+  GoogleAuthOptions,
+  JWT,
+  OAuth2Client,
+  UserRefreshClient,
+} from 'google-auth-library';
+import {ProjectIdCallback} from 'google-auth-library/build/src/auth/googleauth';
```

Comment:
Could I trouble you to go back into google-auth-library, and land a PR to export this properly?